core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -3859,6 +3868,8 @@ private static String javadocLink(String fullyQualifiedClassname) {
     public static final String SECURITY_LOGIN_IMPERSONATION_USERNAME =
         "alluxio.security.login.impersonation.username";
     public static final String SECURITY_LOGIN_USERNAME = "alluxio.security.login.username";
+    public static final String AUTHENTICATION_STALE_CHANNEL_PURGE_INTERVAL =
+        "alluxio.security.purge.interval";
```

Comment:
Could you make the configuration name more explicit?

Reply:
I thought about it and came up with the same name again :)
Any suggestions?

Reply:
my concern is the current name is too general, it could be a purge interval for credentials for instance. Maybe `alluxio.security.stale.channel.purge.interval` or some version of that which does not have as many dots.